### ISSUE TITLE
[FEATURE] Redimensionnement de la fenêtre principale

### ISSUE THREAD

**Issue by Aioniostheos** [MaximilianTyan/GaliTime_Photomaton#3]
La fenêtre principale (ScreenWindow) déborde sur le deuxième écran. Malgré un redimensionnement manuel, lorsque les widgets se rechargent, la fenêtre déborde à nouveau.

Cela peut être corrigé en modifiant la taille maximum de l'affichage central de 1920x1080 en 1280x1024 (dimensions de l'écran).

**Comment by MaximilianTyan** [comment]
Essaie de mettre la fenêtre en plein écran avec F11

**Comment by Aioniostheos** [comment]
Le mode plein écran fonctionne correctement. Merci pour l'info.

**Comment by Aioniostheos** [comment]
Lorsque la vidéo se recharge (passage par la mire), le mode plein écran bloque en 16:9 alors que l'écran principal est en 5:4.

**Comment by MaximilianTyan** [comment]
Tu parles du clear du fichier movie.mjpg toutes les 30 secondes ?

**Comment by Aioniostheos** [comment]
Oui. A chaque fois qu'un widget se recharge, la fenêtre bascule en 16:9.
